WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/settings/WooPosSettingsScreen.kt

```diff
@@ -27,15 +28,31 @@ import com.woocommerce.android.ui.woopos.home.WooPosProductInfoDialog
 import com.woocommerce.android.ui.woopos.home.scanningsetup.WooPosScanningSetupDialog
 import com.woocommerce.android.ui.woopos.root.navigation.WooPosNavigationEvent
 import com.woocommerce.android.ui.woopos.settings.categories.WooPosSettingsCategoriesPaneScreen
+import com.woocommerce.android.ui.woopos.settings.categories.WooPosSettingsCategory
 import com.woocommerce.android.ui.woopos.settings.details.WooPosSettingsDetailPaneScreen
+import kotlinx.coroutines.delay
 
 @Composable
 fun WooPosSettingsScreen(
     onNavigationEvent: (WooPosNavigationEvent) -> Unit,
+    initial: Pair<WooPosSettingsCategory, WooPosSettingsDetailDestination>? = null,
```

Comment:
```suggestion
    destination: Pair<WooPosSettingsCategory, WooPosSettingsDetailDestination>? = null,
```
`initial` is not intuitive, how about `destination`, `initialState`, etc?

Reply:
Good point. Won't change it yet as maybe I need to remove it complete if we go with opening the setup flow dialog from the empty state cart

Reply:
Removed this too